### ISSUE TITLE
nxos_static_routes: CLI error 'context vrf default' on N9K

### ISSUE THREAD

**Issue by allenrobel** [ansible-collections/cisco.nxos#231]
##### SUMMARY

When a vrf is not specified, nxos_static_routes tries to create vrf 'default' on N9K by issuing 'vrf context default'.   This results in a CLI error and the task fails.   If a vrf _is_ specified, the task completes without error.

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME

nxos_static_routes

##### ANSIBLE VERSION

```
ansible 2.10.3
  config file = /Users/arobel/.ansible.cfg
  configured module search path = ['/Users/arobel/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /Users/arobel/ansible-latest/lib/python3.9/site-packages/ansible
  executable location = /Users/arobel/ansible-latest/bin/ansible
  python version = 3.9.0 (default, Oct 27 2020, 14:15:17) [Clang 12.0.0 (clang-1200.0.32.21)]
```

##### CONFIGURATION

```
CACHE_PLUGIN_TIMEOUT(/Users/arobel/.ansible.cfg) = 86400
DEFAULT_LOG_PATH(/Users/arobel/.ansible.cfg) = /tmp/ansible.log
HOST_KEY_CHECKING(/Users/arobel/.ansible.cfg) = False
```

##### OS / ENVIRONMENT

N9K-C9336C-FX2
NXOS:  9.3(6)

##### STEPS TO REPRODUCE

Run the provided task below.

```yaml

    tasks:
    -   cisco.nxos.nxos_static_routes:
            config:
            -   address_families:
                -   afi: ipv4
                    routes:
                    -   dest: 1.1.0.0/24
                        next_hops:
                        -   forward_router_address: 15.1.1.1
                            interface: Ethernet1/1
            state: merged
        name: nxos_static_routes reproduce CLI error
```

##### EXPECTED RESULTS

The task should complete successfully and static route should be created in the global/default vrf.

##### ACTUAL RESULTS

Since, on N9K, 'vrf context default' does not show up in the running-config, nxos_static_routes tries to create vrf default.  This fails with the following CLI error:

```
switch(config)# vrf context default
ERROR: Configuration of VRF default not allowed
switch(config)# 
```

Below is the relevant nxapi-server log

```
2021 January 20 07:22:25.535 pterm_write_to_vsh:527 pid:4514 In vsh [16] Writing cmd "vrf context default"
2021 January 20 07:22:25.568 pterm_write_to_vsh:603 pid:4514 Cmd 'vrf context default' returned with '500'
2021 January 20 07:22:25.568 pterm_write_to_vsh:606 pid:4514 Cmd 'vrf context default' had failure '500'
2021 January 20 07:22:25.568 _ins_api_cli_cmd_conf:621 pid:4514 Failed for cmd: vrf context default code 500
```

**Comment by NilashishC** [comment]
@allenrobel We believe #243 should have fixed this issue. Can you please confirm? Thanks!

**Comment by allenrobel** [comment]
@NilashishC  Hi.  I patched the changes from #243 into a fresh install of cisco.nxos 1.4.0 and this is now working.  Thank you very much for the fix!
